fenixedu-ist-integration/src/main/java/pt/ist/fenixedu/integration/api/FenixAPIv1.java

```diff
@@ -1078,6 +1081,46 @@ public String enrolledByDegreeAndCurricularYear(@PathParam("id") String degreeId
         }).collect(StreamUtils.toJsonArray()).toString();
     }
 
+    @GET
+    @Produces(JSON_UTF8)
+    @Path("users")
+    @OAuthEndpoint(value = PERSONAL_SCOPE, serviceOnly = true)
+    public String getUsers() {
+        Stream<User> users = Bennu.getInstance().getUserSet().stream();
```

Comment:
This was simply a transfer of functionality from the obsolete JerseyServices to service only oauth api. It was working in the past so not much time was spent thinking in optimisations. I'm not against them though.